queue_job/tests/common.py

```diff
@@ -1,9 +1,13 @@
 # Copyright 2019 Camptocamp
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import doctest
+import logging
+import sys
 from contextlib import contextmanager
 
 import mock
 
+# pylint: disable=odoo-addons-relative-import
 from odoo.addons.queue_job.job import Job
```

Comment:
The branch conflicts on this line

Reply:
Fixed!

Reply:
Can you add a comment to explain why this is not a relative import?